html/semantics/scripting-1/the-script-element/script-charset-02.html

```diff
@@ -33,7 +33,7 @@
   <script>
   //these strings should match, since fallback utf-8 is the correct setting.
```

Comment:
This comment is also wrong, but the assert itself is good, but even better would be to replace it with `"\u0139\u203a\xc4\u2021\xc4\u2026\u0139\u013d\u0139\u015f"` which should be what the UTF-8 decodes to if interpreted as windows-1250.

Reply:
should I delete the comment or you suggest.
by the way thanks for guiding me.